kefir.js.flow

```diff
@@ -0,0 +1,158 @@
+/* @flow */
+
+export type Event<V,E> =
+  {type: 'value', value: V} |
+  {type: 'error', value: E} |
+  {type: 'end', value: void};
+
+export type Emitter<V,E> = {
+  value(value: V): boolean;
+  event(event: Event<V,E>): boolean;
+  error(e: E): boolean;
+  end(): void;
+
+  // Deprecated methods
+  emit(value: V): boolean;
+  emitEvent(event: Event<V,E>): boolean;
+};
+
+export type Subscription = {
+  closed: boolean;
+  unsubscribe(): void;
+};
+
+export type Observer<V,E> = {
+  value?: (value: V) => void;
+  error?: (err: E) => void;
+  end?: () => void;
+};
+
+declare class Observable<+V,+E=*> {
+  toProperty(): Property<V,E>;
+  toProperty<V2>(getCurrent: ?() => V2): Property<V|V2,E>;
+  changes(): Observable<V,E>;
+
+  observe(obs: Observer<V,E>): Subscription;
+  observe(onValue: ?(v: V) => void, onError: ?(err: E) => void, onEnd: ?() => void): Subscription;
+  onValue(cb: (v: V) => void): this;
+  offValue(cb: (v: V) => void): this;
+  onError(cb: (err: E) => void): this;
+  offError(cb: (err: E) => void): this;
+  onEnd(cb: () => void): this;
+  offEnd(cb: () => void): this;
+  onAny(cb: (event: Event<V,E>) => void): this;
+  offAny(cb: (event: Event<V,E>) => void): this;
+  log(name?: string): this;
+  offLog(name?: string): this;
+  toPromise(PromiseConstructor?: Function): Promise<V>;
+
+  map<V2>(cb: (v: V) => V2): Observable<V2,E>;
+  filter(cb?: (v: V) => any): Observable<V,E>;
+  take(n: number): Observable<V,E>;
+  takeWhile(cb?: (v: V) => boolean): Observable<V,E>;
+  last(): Observable<V,E>;
+  skip(n: number): Observable<V,E>;
+  skipWhile(cb?: (v: V) => boolean): Observable<V,E>;
+  skipDuplicates(comparator?: (a: V, b: V) => boolean): Observable<V,E>;
+  diff(fn?: (prev: V, next: V) => V, seed?: V): Observable<V,E>;
```

Comment:
Same here with overloads. I just not sure Flow will be able to choose correct overload if both take same arguments.